regolith/tools.py

```diff
@@ -1287,4 +1289,44 @@ def get_id_from_name(coll, name):
                              case_sensitive=False)
     if person:
         return person["_id"]
-    else: return None
\ No newline at end of file
+    else: return None
+
+
+def get_grant_amount(grant, people, begin_date=None, end_date=None):
+    grant_amounts = []
```

Comment:
nice!   I am officially disappointed by two things though.
1) no tests!  Remember, tests first, code after.
2) no docstring.  Remember, docstring comes even before the tests as it forces you to describe in words what the function will do.

Reply:
sorry about that! I have pushed the docstring. the tests are quite tedious to write since there are so many parameters, but I am working on them now! I will take it as an exercise in imagination to pretend I haven't written the code

Reply:
Thanks.  Tests are tedious, until you have been coding for a while and you realize how transformative they are.

Reply:
I have pushed the tests!